src/app/modules/main/wallet_section/transactions/view.nim

```diff
@@ -138,8 +147,33 @@ QtObject:
   proc getChainIdForBrowser*(self: View): int {.slot.} =
     return self.delegate.getChainIdForBrowser()
 
-  proc getLatestBlockNumber*(self: View, chainId: int): string {.slot.} =
-    return self.delegate.getLatestBlockNumber(chainId)
+  proc getLatestBlockNumber*(self: View, chainId: int): int {.slot.} =
+    if self.latestBlockNumbers.hasKey(chainId):
+      let blockData = self.latestBlockNumbers[chainId]
+      let timeDiff = (now() - blockData.datetime)
+      var multiplier = -1.0
+      if blockData.isLayer1:
+        multiplier = 0.083 # 1 every 12 seconds
+      else:
+        case chainId:
+          of 10: # Optimism
```

Comment:
I think we have constants defined for all the chains we use in a very common spot

Reply:
Huh did a quick search, seems like we don't... We should!
Theres a even a `wallet_constants` file